js/src/block-editor/components/form-controls.js

```diff
@@ -1,25 +0,0 @@
-/**
- * Internal dependencies
- */
-import { Fields } from './';
-
-/**
- * Gets the form controls for a block.
- *
- * @param {Object} props The props of this components.
- * @param {Object} props.blockProps The block's props.
- * @param {Object} props.block The block.
- * @return {Function|null} The Edit component for the block.
- */
-const FormControls = ( { blockProps, block } ) => {
-	return (
-		<Fields
```

Comment:
This file is deleted because it's not really needed, it's just a wrapper for `<Fields>`